packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl

```diff
@@ -1,126 +1,109 @@
-# Customize inbound failsafe rules
-# https://docs.projectcalico.org/v3.5/getting-started/bare-metal/policy/failsafe
+# General
+# =======
+---
+# Disable default failsafe inbound rules
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  failsafeInboundHostPorts: []
```

Comment:
I think the next improvement (out of scope for this PR) we do, can be to improve the FelixConfiguration we use. What do you think?

If you agree, should we create an issue about it?

Reply:
You can always create an issue for something you think needs some work.